vllm/v1/engine/core.py

```diff
@@ -17,7 +17,7 @@
     maybe_register_config_serialize_by_value)
 from vllm.utils import get_exception_traceback, zmq_socket_ctx
 from vllm.v1.core.kv_cache_utils import get_kv_cache_config
-from vllm.v1.core.scheduler import Scheduler
+from vllm.v1.core.scheduler.scheduler import Scheduler
```

Comment:
Is the intent here to also update the engine core to use a configurable type for the scheduler, now that we have an interface for it?